src/Entity/RateCard.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\RateCardRepository")
+ */
+class RateCard
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $solution;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $prestation;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $models;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $priceRateCard;
+
+    /**
```

Comment:
RateCard n'est pas lié à Factures

Reply:
Je pense que si unique index voudrai dire one to one  -> CREATE TABLE rate_card (id INT AUTO_INCREMENT NOT NULL, facture_id INT DEFAULT NULL, solution VARCHAR(255) NOT NULL, prestation VARCHAR(255) NOT NULL, models VARCHAR(255) NOT NULL, price_rate_card INT NOT NULL, UNIQUE INDEX UNIQ_E9102ADC7F2DEE08 (facture_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB
et pour simulation      -> CREATE TABLE options_simulation (options_id INT NOT NULL, simulation_id INT NOT NULL, INDEX IDX_939E79CE3ADB05F1 (options_id), INDEX IDX_939E79CEFEC09103 (simulation_id), PRIMARY KEY(options_id, simulation_id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB
J'ai bien créer la bidirectionnalité ..